gon/shaped.py

```diff
@@ -89,9 +84,10 @@ def __contains__(self, point: Point) -> bool:
         >>> Point(7, 0) in polygon
         False
         """
-        return (point_in_polygon(point.raw(),
-                                 (self._raw_border, self._raw_holes))
-                is not PointLocation.EXTERNAL)
+        raw = self._raw_border, self._raw_holes
```

Comment:
Why don't we have it as a property?

Reply:
we have, but it makes copies of `holes` list so the user can mutate it safely, but I don't need to create copies so we are saving memory here